### ISSUE TITLE
Suggestion: Include profile name in log file (or job queue)

### ISSUE THREAD

**Issue by vonH** [staxrip/staxrip#331]
**Is your feature request related to a problem? Please describe.**
In nearly all previous versions of Staxrip, the profile name has been present in the job queue which has been useful. This is no longer the case in 2.1.3.0. I have checked job Options and General Settings but found no option to enable this functionality.

**Describe the solution you'd like**
I believe the reason to remove the profile name from the job queue was to make the display easier to read on low resolution displays. This is a good reason, but perhaps including it as a text file in the temp files, or even as a single line in the log file would be a great compromise and be very useful. 

Even though the AviSynth scripts are written in the temp files, users who rely on profiles would have to dig through the avs files and the encoding settings in order to work out which profile was used (it's not featured anywhere once the job has been added to the queue).

**Describe alternatives you've considered**
The only alternative is to check the avs files and use mediainfo to check the x264 encoding settings. For example, before encoding I usually make a number of test encodes with SelectEvery() and then full videos. I save these as Source_Video_Name_Month_Test_X, where X=test revision. Some tests may change PsyRD to 1.10 from 1.00, for example.

**Explain why the feature matters to you**
I believe the profile name should be written somewhere (ideally in the log file) as it makes sense to have this information recorded. I have attached a screenshot of a standard log file and one with the proposed changes. This would keep the job queue clean.

**Additional context**
Existing log file:
![image](https://user-images.githubusercontent.com/10042301/92430404-a4c04180-f18c-11ea-8c89-41b38c5d7465.png)
Proposed change:
![image](https://user-images.githubusercontent.com/10042301/92430437-b6094e00-f18c-11ea-9b04-6693c1931251.png)

**Comment by Dendraspis** [comment]
I think we can extend the log with this useful information, but it might take a bit.

**Comment by Dendraspis** [comment]
Template and Profile are shown now in the logfile:

```
...
---------------------------- Configuration ----------------------------

Template : Automatic Workflow
Profile  : Test-Profile
Muxer    : MKV (mkvmerge)

--------------------------- AviSynth Script ---------------------------
...
```
So let's wait for the next beta version.  🙂 

----------
Fixed in (https://github.com/staxrip/staxrip/commit/dd58294b1663a2381439e2d5463f516bcca7465a) and (https://github.com/staxrip/staxrip/commit/262b25ad57e9edb6d831b4b6dd3dbb12935449d1).

**Comment by JJKylee** [comment]
There are three types of _named_ **profile**s in StaxRip: video encoder, audio encoder, and muxer. Exactly which profile is the log referring to?

I guess it's video encoder, right? If so, please clarify that by writing down the correct title explicitly like "Video encoder profile:", etc.

**Comment by Dendraspis** [comment]
Yes, for example *x264*, *x265*, *x265_Medium_Grain*, *TVSeries_Standard*, ...
Maybe I orientated myself too much on the code, where this is the only "Profile".  😄 

> If so, please clarify that by writing down the correct title explicitly like "Video encoder profile:", etc.

You're absolutely right, Sir ....... done by (https://github.com/staxrip/staxrip/commit/65129d7a0872482c7696dbb612bc2eed70314aad).  ✔️ 

```
Template: ...
Video Encoder Profile: ...
Container/Muxer Profile: ...
```

**Comment by vonH** [comment]
> There are three types of _named_ **profile**s in StaxRip: video encoder, audio encoder, and muxer. Exactly which profile is the log referring to?
> 
> I guess it's video encoder, right? If so, please clarify that by writing down the correct title explicitly like "Video encoder profile:", etc.

I meant the feature named "Templates" and should have used the correct terminology (I was in a rush when creating that report). For anything else (codecs, etc.), using mediainfo is the obvious tool to check on encoding options and codecs, chapters, etc.

**Comment by vonH** [comment]
> Template and Profile are shown now in the logfile:
> 
> ```
> ...
> ---------------------------- Configuration ----------------------------
> 
> Template : Automatic Workflow
> Profile  : Test-Profile
> Muxer    : MKV (mkvmerge)
> 
> --------------------------- AviSynth Script ---------------------------
> ...
> ```
> 
> So let's wait for the next beta version. 🙂
> 
> Fixed in ([dd58294](https://github.com/staxrip/staxrip/commit/dd58294b1663a2381439e2d5463f516bcca7465a)) and ([262b25a](https://github.com/staxrip/staxrip/commit/262b25ad57e9edb6d831b4b6dd3dbb12935449d1)).

Thank you for adding this in, and I apologise for using the incorrect terminology. I can easily understand how that caused confusion.
